engine/src/main/java/org/camunda/bpm/engine/management/SetJobRetriesAsyncBuilder.java

```diff
@@ -0,0 +1,124 @@
+   *
+   * @param query the process instance query
+   *
+   * @see ManagementService#setJobRetriesAsync(List, ProcessInstanceQuery, int)
+   *
+   * @return the builder instance
+   */
+  SetJobRetriesAsyncBuilder processInstanceQuery(ProcessInstanceQuery query);
+
+  /**
+   * Specifies a historic process instance query that identifies runtime process instances with jobs that have to be modified. All jobs belonging to those process instances will be updated.
+   *
+   * <strong>Note:</strong> You can reference jobs either by specific jobs ({@link #jobQuery}, {@link #jobIds}) or by process ({@link #processInstanceIds}, {@link #processInstanceQuery}, {@link #historicProcessInstanceQuery}).
+   * If you provide a historicProcessInstanceQuery, calling {@link #jobQuery} or {@link #jobIds} is not allowed and will cause a {@link ProcessEngineException} on {@link #execute()}.
+   *
+   * @param query the historic process instance
+   *
+   * @see ManagementService#setJobRetriesAsync(List, ProcessInstanceQuery, HistoricProcessInstanceQuery, int)
+   *
+   * @return the builder instance
+   */
+  SetJobRetriesAsyncBuilder historicProcessInstanceQuery(HistoricProcessInstanceQuery query);
+
+  /**
+   * Specifies a list of job ids that will be updated.
+   *
+   * <strong>Note:</strong> You can reference jobs either by specific jobs ({@link #jobQuery}, {@link #jobIds}) or by process ({@link #processInstanceIds}, {@link #processInstanceQuery}, {@link #historicProcessInstanceQuery}).
+   * If you provide a list of jobs, calling {@link #processInstanceIds}, {@link #processInstanceQuery} or {@link #historicProcessInstanceQuery} are not allowed and will cause a {@link ProcessEngineException} on {@link #execute()}.
+   *
+   * @param query the query to reference jobs that will be updated.
+   *
+   * @see ManagementService#setJobRetriesAsync(JobQuery, int)
+   *
+   * @return the builder instance
+   */
+  SetJobRetriesAsyncBuilder jobIds(List<String> jobIds);
+
+  /**
+   * Specifies a due date to be set on the referenced {@link Job jobs}.
+   *
+   * When the number of retries of a job are incremented it is not automatically scheduled for immediate execution.
+   * When a {@link Job} is executed is determined by the due date. By setting the due date together with the job retries, the scheduled execution date of the
+   * job can be adjusted.
+   *
+   * @param the new due date for the updated jobs. If it is null, the retries of the jobs will be updated but the due date is ignored.
+   *
+   * @return the builder instance
+   */
+  SetJobRetriesAsyncBuilder dueDate(Date dueDate);
+
+  /**
+   * Closes the fluent builder and executes the instructions.
+   *
+   * @throws ProcessEngineException Check the builder methods and their linked counterparts in {@link ManagementService} for more detail
+   */
+  Batch execute();
```

Comment:
```suggestion
  Batch executeAsync();
```
Let's streamline the method names between batch operation fluent builders.